docs/airdrop-eligibility-distribution.md

```diff
@@ -216,31 +218,42 @@ Scroll to the section that describes your scenario and follow the instructions.
-5. Now you should have token totals, calculated based on points scored before and after Nitro. Calculate the difference between the two token totals, halve that, and add it to the token total calculated based on `PointsPre`. This is your final amount of eligible tokens.
-6. In the case that your token total based on `PointsPost` was lower than the other total, you were awarded tokens solely based on `PointsPre`, as it was the higher amount.
- 2. Next, refer to the [point-to-token conversion table](#point-to-token-conversion-table) and determine how many tokens you're eligible to claim based on the number of points you earned.
+ 1. First, refer to the [qualifying actions section](#qualifying-actions) to determine how many points you earned **before** Arbitrum Nitro launched on Arbitrum One mainnet. We'll refer to this as `OnePointsPre`.
+ 2. If you performed any qualifying actions on Arbitrum Nova before Nitro was launched, reference the second step of `Scenario 2` above to determine `NovaPointsPre`.
+ 3. Next, TODO. We'll refer to this total as `PointsPre`.
+ 4. Use the [point-to-token conversion table](#point-to-token-conversion-table) to convert `PointsPre` into tokens. We'll refer to this total as `TokensPre`.
+ 5. Then, refer to the [qualifying actions section](https://github.com/ArbitrumFoundation/docs/pull/204#qualifying-actions) to determine how many points you earned **after** Nitro launched on Arbitrum One. We'll refer to this as `OnePointsPost`.
+ 6. If you performed any qualifying actions on Arbitrum Nova after Nitro was launched, reference the second step of `Scenario 2` above to determine `NovaPointsPost`.
+ 7. Next, TODO. We'll refer to this total as `PointsPost`.
+ 8. Use the [point-to-token conversion table](https://github.com/ArbitrumFoundation/docs/pull/204#point-to-token-conversion-table) to convert your `PointsPost` into tokens. We'll refer to this total as `TokensPost`.
+ 9. Subtract `TokensPre` from `TokensPost`. If the result is a positive number, divide it by 2 and add it to `TokensPre`. This is your `TokenEntitlement` - the number of tokens that you're entitled to claim.
 
-todo
 
-Note that the minimum airdrop entitlement is **625** tokens; the maximum airdrop entitlement is **10,250** tokens.
+**Examples:**
+
+| `OnePointsPre` | `NovaPointsPre`   | `PointsPre` | `TokensPre` | `OnePointsPost` | `NovaPointsPost`  | `PointsPost` | `TokensPost` | `TokenEntitlement` |
+| -------------- | ----------------- | ----------- | ----------- | --------------- | ----------------- | ------------ | ------------ | ------------------ |
+| 5              | 2 (gets cut to 1) | 6           | 3,250       | 5               | 2 (gets cut to 1) | 6            | 3,250        | 4,750              |
+| 2              | 2                 | 4           | 1,750       | 7               | 2 (gets cut to 1) | 8            | 4,250        | 3,000              |
+
+
+TODO - verify that the above is correct
```

Comment:
todo @pearring - can you spot-check these examples - are the calculations correct?